frontend/src/metabase/api/task.ts

```diff
@@ -0,0 +1,34 @@
+import type {
+  ListTasksRequest,
+  ListTasksResponse,
+  Task,
+} from "metabase-types/api";
+
+import { Api } from "./api";
+
+export const taskApi = Api.injectEndpoints({
+  endpoints: builder => ({
+    listTasks: builder.query<ListTasksResponse, ListTasksRequest>({
+      query: params => ({
+        method: "GET",
+        url: "/api/task",
+        params,
+      }),
+    }),
+    getTask: builder.query<Task, number>({
+      query: id => ({
+        method: "GET",
+        url: `/api/task/${id}`,
+      }),
+    }),
+    getTasksInfo: builder.query<unknown, void>({
+      query: () => ({
+        method: "GET",
+        url: "/api/task/info",
+      }),
+    }),
```

Comment:
I did not type this now because this point was not used by the entity. We can introduce proper types once we start replacing the `services.js` APIs in the milestone 2 of #40788